Dockerfile

```diff
@@ -57,6 +60,8 @@ COPY --from=app-builder --chown=skate:skate /root/priv/static ./priv/static
 # Add application artifact compiled in app-builder container
 COPY --from=app-builder --chown=skate:skate /root/_build/prod/rel/skate .
 
+ADD --chown=skate:skate /root/aws-cert-bundle.pem ./priv/aws-cert-bundle.pem
```

Comment:
suggestion: you want to `COPY` this from the builder, instead of `ADD`ing it from the local filesystem.